tests/manager/manager-executeCommand-004.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+MongoDB\Driver\Manager::executeCommand() options
+--SKIPIF--
+<?php if (getenv("TRAVIS")) exit("skip This currently tails on Travis because it doesn't run 3.6 yet"); ?>
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php NEEDS('STANDALONE'); CLEANUP(STANDALONE); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+require_once __DIR__ . "/../utils/observer.php";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+(new CommandObserver)->observe(
+    function() use ($manager) {
+        $command = new MongoDB\Driver\Command([
+            'ping' => true,
+        ]);
+
+        try {
+            $manager->executeCommand(
+                DATABASE_NAME,
+                $command,
+                [
+                    'readPreference' => new \MongoDB\Driver\ReadPreference(\MongoDB\Driver\ReadPreference::RP_SECONDARY),
+                    'readConcern' => new \MongoDB\Driver\ReadConcern(\MongoDB\Driver\ReadConcern::LOCAL),
+                    'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY),
+                ]
+            );
+        } catch ( Exception $e ) {
+            // Ignore exception that ping doesn't support writeConcern
+        }
+    },
+    function(stdClass $command) {
+        echo "Read Preference: ", $command->{'$readPreference'}->mode, "\n";
+        echo "Read Concern: ", $command->readConcern->level, "\n";
+        echo "Write Concern: ", $command->writeConcern->w, "\n";
+    }
+);
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+Read Preference: secondary
+Read Concern: local
```

Comment:
Note: this test currently fails because [CDRIVER-2418](https://jira.mongodb.org/browse/CDRIVER-2418) now ignores the `readConcern` option unless the read flag is set for the command (not done for the raw and write modes). This should be fixed by [CDRIVER-2422](https://jira.mongodb.org/browse/CDRIVER-2422).